src/plugins/discover/public/application/angular/context/api/utils/get_es_query_sort.ts

```diff
@@ -14,11 +14,21 @@ import { EsQuerySortValue, SortDirection } from '../../../../../kibana_services'
  * @param timeField
  * @param tieBreakerField
  * @param sortDir
+ * @param nanos
  */
 export function getEsQuerySort(
   timeField: string,
   tieBreakerField: string,
-  sortDir: SortDirection
+  sortDir: SortDirection,
+  nanos?: string
 ): [EsQuerySortValue, EsQuerySortValue] {
-  return [{ [timeField]: sortDir }, { [tieBreakerField]: sortDir }];
+  return [
+    {
+      [timeField]: {
+        order: sortDir,
+        format: nanos ? 'strict_date_optional_time_nanos' : 'strict_date_optional_time',
```

Comment:
Couldn't the format be empty for the **non nanos** case?

Reply:
It works without it, but Jim suggested we always send a format:
https://elastic.slack.com/archives/C017U8Q558X/p1618228972049700?thread_ts=1617866399.044000&cid=C017U8Q558X

Reply:
I would try to keep the request more compact. 
But as you prefer!